src/i18n/i18n.ts

```diff
@@ -182,15 +183,19 @@ const dict = {
         denomination_tooltip: "Choose your preferred denomination: BTC or sats",
         decimal_tooltip:
             "Choose your preferred decimal separator: dot or comma",
-        claim_success: "Swap {{ id }} claimed successfully!",
+        swap_completed: "Swap {{ id }} completed successfully!",
         claim_fail: "Failed to claim swap: {{ id }}",
         logs: "Logs",
-        logs_tooltip: "Logs of the web app, useful for debugging.",
-        enable_audio_notifications: "Enable Audio Notifications",
+        logs_tooltip: "Logs of the web app, useful for debugging",
+        enable_audio_notifications: "Audio Notifications",
         enable_audio_notifications_tooltip:
             "Enable or disable audio notifications",
         on: "on",
         off: "off",
+        notification_header: "Boltz Swap",
```

Comment:
```suggestion
        notification_header: "Boltz",
```
I'd call this just Boltz. It's cleaner

Reply:
Cleaned. For all languages.